pinecone/data/import_error.py

```diff
@@ -0,0 +1,30 @@
+class IndexClientInstantiationError(Exception):
```

Comment:
Since I was already in a mode of pulling things out of `index.py`, I decided to move this to a different file as well.